### DISCUSSION TITLE
772

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#942] [Comments]
# 772

요즘 데이식스의 "한 페이지가 될 수 있게" 노래에 엄청 꽂혔다. 정확히는 꽂힌 가사가 있다. "아름다운 청춘의 한 장 함께 써내려 가자 너와의 추억들로 가득 채울래 지금 이 순간이 다시 넘겨볼 수 있는 한 페이지가 될 수 있게" 어떤 동시대를 함께 보내는 모든 그룹 혹은 조직의 일원들간에도 통하는 이야기 같았다. 특히 스타트업 안에서 일하는 동료들간에도, 조직과 팀원 사이에도 통하는 이야기인 것 같았다. 대학교때는 학과 생활보다는 동호회 생활을 동기들과 재밌게 했다. 졸업한 후에도 계속 연락하고 만날 정도로 오랜 인연을 이어가고 있다. 동기들과 동호회 활동을 하면서 이런 저런 행사도 준비하고 대회도 나가고 늦게까지 술도 마시고, 자취방에서 게임도 하고 그랬던 20대 이야기는 지금도 만나면 자주 꺼낸다...

https://jojoldu.tistory.com/772

**Comment by solarrrrr1010** [comment]
함께 쌓아나가고 싶습니다.
----------------
주니어 개발자입니다. 개발 초기 데이식스 알게 돼서 지금껏 듣고 있습니다.
